src/pipecat/services/ai_services.py

```diff
@@ -278,6 +282,11 @@ async def cancel(self, frame: CancelFrame):
             await self._stop_frame_task
             self._stop_frame_task = None
 
+    async def process_frame(self, frame: Frame, direction: FrameDirection):
+        super().process_frame(frame, direction)
```

Comment:
We are still missing the `await` here.

Reply:
Egads. Sorry. I clearly do not know how to use computers.